packages/next/src/server/app-render/app-render.tsx

```diff
@@ -2935,6 +3004,11 @@ async function prerenderToStream(
           implicitTags,
           renderSignal: initialClientController.signal,
           controller: initialClientController,
+          // TODO we don't really need to scope a cache signal for
+          // client prerenders because there are no APIs which need
+          // access to this signal during SSR. Module loading does track
+          // in progress loads using a cache signal but that is referenced
+          // from module scope.
```

Comment:
I agree with this assessment. Should we just set the `cacheSignal` to null then?